### ISSUE TITLE
Question about MatterGen training

### ISSUE THREAD

**Issue by pjh1921** [microsoft/mattergen#90]
Thank you for your great work on MatterGen! I really appreciate the effort you've put into this project.
I have a couple of questions regarding the training process:
1. Appropriate loss_val range
Instead of using alex-mp-20 or mp-20, I am currently training the model with my own dataset, which consists of 26,000 data points. I would like to know what range of loss_val would be considered appropriate or reasonable during training.
Previously, when I ran a test training with 12000 data points , the checkpoint in the lightning_logs directory was named: epoch=169-loss_val=2.44.ckpt
I am wondering if this loss_val is within a reasonable range. Additionally, if further improvements can be made, what methods should I consider to optimize the training process?

2. Difference between checkpoints/ and lightning_logs/ checkpoints in the training output directory
I noticed that the training output directory contains a checkpoints/ directory as well as checkpoints inside lightning_logs/. Could you clarify the difference between these two?

Any insights or recommendations would be greatly appreciated.
Thank you in advance!

**Comment by danielzuegner** [comment]
Hi @pjh1921,

1. Unfortunately the loss values between different datasets cannot be directly compared. For example, the position loss per structure is a sum of the individual losses per atom -- so if the number of atoms per structure grows, the loss will grow proportionally. Therefore, I can't give you a concrete loss value your model should achieve. In the end, it comes down to the quality of the generated structures. I would suggest you monitor both training and validation loss to see whether the model is overfitting -- if validation loss is much higher than training loss, this indicates overfitting.
2. I recommend you use the checkpoints located in `lightning_logs` -- while they are equivalent in principle, `lightning_logs` also contains `last.ckpt`, i.e., the last checkpoint saved, giving you more options when loading the model.
